pkg/util/constants.go

```diff
@@ -76,7 +79,10 @@ func ConstantWithMetamorphicTestValue(name string, defaultValue, metamorphicValu
 }
 
 // rng is initialized to a rand.Rand if crdbTestBuild is enabled.
-var rng *rand.Rand
+var rng struct {
+	r *rand.Rand
```

Comment:
nit: can't we embed this one too?

Reply:
Figured making the caller reach for the inner field was a better way to signal you needed to lock the containing struct first